webapp/channels/src/components/sidebar/channel_filter/channel_filter.tsx

```diff
@@ -101,10 +86,21 @@ export class ChannelFilter extends React.PureComponent<Props> {
                     >
                         <i className='icon icon-filter-variant'/>
                     </a>
-                </OverlayTrigger>
+                </WithTooltip>
             </div>
         );
     }
 }
 
+const messages = defineMessages({
```

Comment:
This is a me thing, so feel free to ignore.

I like the order of things in the component file to be:
- Imports
- Extra stuff needed (messages, constants, types...)
- Component
- Export

That way, when you are looking into the component, you have already seen "everything", and there is nothing "hidden after the component" (usually the component is the biggest thing in the file).

Therefore, I would move this up before the component (and similar in other files where you have done the same).

But... this is me wanting things to be done my way XD Happy to leave it as this if you feel it is better for any reason.

Reply:
My thinking was that I wanted them to be after the component because I want the component to be the first thing that I see when opening the file, and it'll get pushed off the screen if there's more than a few messages. That said, it is kind of arbitrary to separate them from other constants that way, and I also quickly looked through everywhere that we currently use `defineMessages`, and I couldn't find a single place we defined them after the component. I'll go fix that in another force push so that I can keep the commit history clean for the campaign.

Thanks for pointing that out since I thought that was a good idea at the time, but consistency is more important than slightly ordering things differently

Reply:
The best about consistency is that if you are faster than the rest, you are able to get your own styles rules applied "just for consistency" :rofl: